src/fixit/cli.py

```diff
@@ -349,3 +349,51 @@ def debug(ctx: click.Context, paths: Sequence[Path]) -> None:
             "disabled:",
             sorted(f"{rule()} ({reason})" for rule, reason in disabled.items()),
         )
+
+
+@main.command()
+@click.pass_context
+@click.argument("path", nargs=1, type=click.Path(exists=True, path_type=Path))
+def validate_config(ctx: click.Context, path: Path) -> None:
+    """
+    validate the config(s) for the provided path(s)
+    """
+    try:
+        from rich import print as pprint
+    except ImportError:
+        from pprint import pprint  # type: ignore
+
+    exception_raised = False
+
+    try:
+        configs = read_configs([path])[0]
+
+        def validate_rules(rules: List[str], path: Path) -> None:
+            for rule in rules:
+                try:
+                    qualified_rule = parse_rule(rule, path, configs)
+                    for _ in find_rules(qualified_rule):
+                        pass
+                except Exception as e:
+                    nonlocal exception_raised
+                    exception_raised = True
+                    pprint(
+                        f"Failed to parse rule `{rule}`: {e.__class__.__name__}: {e}"
+                    )
```

Comment:
Can we have separate try/except blocks and error messages for "parse failure" and "import failure"?

Would also be cool if `validate_rules` could take an extra context string, so that the error messages could reference what overrides block/path they were in relation to?